src/main/java/seedu/contax/model/appointment/Appointment.java

```diff
@@ -65,6 +89,14 @@ public Person getPerson() {
         return person;
     }
 
+    public Priority getPriority() {
+        return priority;
+    }
+
+    public Appointment setPriority(Priority priority) {
```

Comment:
You should change the name to `withPriority` to reflect that this object is immutable.

Reply:
thx will change